data-prepper-main/src/main/java/org/opensearch/dataprepper/ContextManager.java

```diff
@@ -26,6 +26,14 @@ public class ContextManager {
 
     private final AnnotationConfigApplicationContext coreApplicationContext;
 
+    private static final String DATA_PREPPER_HOME = System.getProperty("user.dir");
+    private static final String PIPELINES_CONFIG = DATA_PREPPER_HOME + "/pipelines/pipelines.yaml";
```

Comment:
For loading multiple pipeline files (#1736 ). I can change `PIPELINES_CONFIG` point to the `pipelines/` directory.